package.json

```diff
@@ -12,5 +12,13 @@
   },
   "devDependencies": {
     "husky": "^8.0.1"
+  },
+  "dependencies": {
+    "@emotion/react": "^11.10.6",
+    "@emotion/styled": "^11.10.6",
+    "@material-ui/core": "^4.12.4",
+    "@mui/material": "^5.11.11",
+    "bootstrap": "^5.2.3",
+    "react-bootstrap": "^2.7.2"
```

Comment:
You are not supposed to add any dependencies in the root folder. Only inside the ui folder.